code/style.css

```diff
@@ -138,3 +146,51 @@ button:hover {
   opacity: 0.9;
   transition: all 0.2s ease;
 }
+
+.sweet {
+  background-color: rgb(207, 125, 200);
+  font-size: 22px;
+}
+
+.sour {
+  background-color: rgb(177, 85, 128);
+  font-size: 22px;
+}
+
+.salty {
+  background-color: rgb(116, 38, 105);
+  font-size: 22px;
+}
+
+.hg {
+  background-color: rgb(207, 125, 200);
+  font-size: 22px;
+}
+
+.yes {
+  background-color: rgb(207, 125, 200);
+  font-size: 22px;
+}
```

Comment:
This class is exactly the same as `.sweet` 🍭 🍬 . It would be better to create classes like `.opption1` to `.option4` and reuse it 😸 🤓